views/backend/settings.php

```diff
@@ -35,58 +28,23 @@
         </table>
 
     </div>
+
     <div class="col m6 s12">
         <h4>Ajouter un modo</h4>
 
-        <?php
-            if(filter_has_var(INPUT_POST, 'submit')){
-                if(filter_has_var(INPUT_POST, 'name')){
-                    $name = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'name')));
-                }
-                if(filter_has_var(INPUT_POST, 'email')){
-                    $email = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'email')));
-                }
-                if(filter_has_var(INPUT_POST, 'email_again')){
-                    $email_again = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'email_again', FILTER_VALIDATE_EMAIL)));
-                }
-                if(filter_has_var(INPUT_POST, 'role')){
-                    $role = filter_var(htmlspecialchars(filter_input(INPUT_POST, 'role')));
-                }
-                
-                $token = $model_user->token(30);
-
-                $errors = [];
-
-                if(empty($name) || empty($email) || empty($email_again)){
-                    $errors['empty'] = "Veuillez remplier tous les champs";
-                }
-
-                if($email != $email_again){
-                    $errors['different'] = "Les adresses email ne correspondent pas";
-                }
-
-                if($model_user->email_taken($email)){
-                    $errors['taken'] = "L'adresse email est déjà assignée à un modérateur";
-                }
-
-                if(!empty($errors)){
+                   <?php if(!empty($errors)){
                     ?>
                         <div class="card red">
                             <div class="card-content white-text">
                                 <?php
                                 foreach($errors as $error){
-                                    echo $error;
+                                    echo $error."</br>";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$error'.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)